### DISCUSSION TITLE
How can I capture something to tomorrow daily note?

### DISCUSSION THREAD

**Discussion by oldflag2333333** [chhoumann/quickadd#278] [Q&A] [answered]
I want to create some next day tasks to tomorrow daily note, but I don't know how to get it. Anybody some ideas?

**Comment by FelipeRearden** [comment, marked as answer]
Hello @oldflag2333333 !!!

From the QuickAdd docs, we have this options:
![6A907C16-6073-4F78-8F69-606C5D77DAA8](https://user-images.githubusercontent.com/85180696/187899661-758cdb55-3b48-4d0f-bfcf-4b684cc24c8b.jpeg)

https://github.com/chhoumann/quickadd/blob/master/docs/FormatSyntax.md

Maybe this would help you :)

Have a great day !

**Reply by oldflag2333333** [reply]
Thanks, that's what I want

**Comment by oldflag2333333** [comment]
Thanks, that's what I want
